src/main/java/com/google/firebase/auth/Utils.java

```diff
@@ -0,0 +1,19 @@
+package com.google.firebase.auth;
+
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.base.Strings;
+
+public class Utils {
```

Comment:
This class has to package-protected. If it must be public, then it should be moved to the `auth.internal` package.

Reply:
Done.